tests/integration/targets/zypper/tasks/zypper.yml

```diff
@@ -442,3 +442,17 @@
     name: gnu-netcat
     state: present
     force_resolution: True
+
+- name: remove gnu-netcat and re-install netcat-openbsd
+  zypper:
+    name:
+      - -gnu-netcat
+      - +netcat-openbsd
+    state: present
+
+- name: Install gnu-netcat with force_resolution and replacefiles
+  zypper:
+    name: gnu-netcat
+    state: present
+    force_resolution: true
+    replacefiles: true
```

Comment:
This is not a good test of replacefiles. The force_resolution has been tested above, so I am not sure those two tasks bring anything. Ideally we should craft a fixture with two non-conflicting rpms with conflicting files, copy and install them with replace files, and check the results.

Reply:
59c7449 includes the above. But the tests "install netcat-openbsd which conflicts with gnu-netcat" fails. 
Are test environments re-used? Looks like a previous run left some packages behind causing an unexpected failure in the test.

Reply:
Maybe we should just add a removal of those packages first, to make sure we are clean.

Reply:
Done in b12916b